data/map planets.txt

```diff
@@ -2504,6 +2542,23 @@ planet Luna
 		threshold 7500
 		fleet "Large Republic" 10
 
+planet "M.B. Station"
+	attributes CCOR military station urban
+	landscape land/city6
+	description "Mordente-Bridi Station, commonly shortened to M.B. Station, is a testiment to the engineering corps of the now defunct CCOR. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest armor CCOR could manufacture at scale. Bridi Cylinder is pressurized containing crew quarters and logistical infrastructure, amounting to a significant city, while the Mordente Cylinder contains what was CCOR's main shipyard."
+	spaceport "Almost none of the Mordente Cylinder is pressurized. Within the armored hull is a micro-gravity shipyard complete with worker-shuttles busily zipping about constructing and repairing a plethora of ships and the fighter hangars in addition to the spaceport."
```

Comment:
```suggestion
planet Mordente-Bridi
	attributes CCOR military station
	landscape land/city6
	description `A testament to the engineering corps of the now defunct CCOR, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest armor the CCOR could manufacture at scale. Bridi Cylinder is pressurized containing crew quarters and logistical infrastructure, amounting to a significant city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
	spaceport `Apart from the spaceport facilities, almost none of the Mordente Cylinder is pressurized. Making up most of its armored hull is a micro-gravity shipyard, which despite not being subject to a central government and defense force has remained quite busy. Worker shuttles are busily flying about the large space, tending to a multitude of ships; you notice that a few of them are also engaged in constructing new ships.`
```

`urban` is generally restricted to space objects with > 1 billion inhabitants. e.g. Solace has `urban` (3 billion inhabitants), Zug does not (500 million inhabitants).

Reply:
I don't know what restrictions we may have for high population numbers but Mordente-Bridi is meant to be an O'Neil Cylinder. It's internal living space would be comparable to a continent. I think I may've been underselling it with the addition of 'amounting to a significant city' in my last edit.

Other than that I'd prefer that Mordente-Bridi keep Urban the adjustments to the description and spaceport are great. Maybe adjust 'amounting to a large city' to something analogous to 'comparable to a densely populated planet' if we keep Urban.

Reply:
I was under the impression that M.B. was the O'neil cylinder.

Reply:
Are M.B. and Mendez supposed to be the same type of station or similar size? I feel like it would be better if one was larger than the other.

Reply:
Mordente-Bridi is quite a bit larger. Mendez is not intended to be an O'Neil Cylinder, just a ramscoop station refinery. It collects solar wind to turn into fuel but little, if any, more.

Reply:
Ah, alright. I think I misread Mendez being one.

Reply:
Committing bene's suggestion for now while retaining the urban tag, can edit it if needed.